scripts/build/nightly_build.sh

```diff
@@ -27,6 +27,8 @@ sudo apt-get install -y unzip python3-h5py libhdf5-dev libio-socket-ssl-perl  li
 cd ${HOME}
 wget https://drive.google.com/uc\?export\=download\&id\=1ythlWTtOCqm2nSKQzvc6QLiFYeFgC1NX -O mmg.zip
```

Comment:
can you add comments what it is and what it is being used for?
makes it easier to keep the overview once more things are added

Reply:
OK, one question, the MKL licence allows to include it?

Reply:
Well we are just compiling, I assume we can

Reply:
hm afaik its not a problem but to be sure we should ask @armingeiser and @oberbichler

Reply:
Should be fine. AFAIK it should also not be a problem to include Eigen as an external library inside the repo as long as all **modifications on Eigen** are disclosed undered MPL (but I am no licencing expert).

Are there any differences between the files on Google Drive and the official download?

Reply:
I try to sum up, but also I am no licensing expert. There are two different licenses involved:

* Eigen: which is mostly MPL2 and (last time i checked) some parts that are LGPL. But that can be exluded with a compiler flag - and we are doing that.

* Intel MKL has its own license, but it is for free and everyone that registers at the intel page can download and use it. To my understanding some parts (the redistributables) can even be distributed e.g. in the release.

Reply:
It seems that as long as we are not modifying something inside Eigen we are safe.